package.json

```diff
@@ -12,9 +12,9 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
-    "@edx/paragon": "^0.0.1",
+    "@edx/paragon": "git+https://github.com/edx/paragon.git#9f1c456bc727c05042ee3da0b2330937702d011b",
```

Comment:
Reviewers: I'll update this to an official release once https://github.com/edx/paragon/pull/55 has merged and the package has been published to npm.